lib/jekyll-github-metadata/metadata_drop.rb

```diff
@@ -23,6 +23,7 @@ def keys
       def_delegator :"Jekyll::GitHubMetadata::Pages", :pages_hostname, :pages_hostname
       def_delegator :"Jekyll::GitHubMetadata::Pages", :api_url, :api_url
       def_delegator :"Jekyll::GitHubMetadata::Pages", :help_url, :help_url
+      def_delegator :"Jekyll::GitHubMetadata::Pages", :mention_url_base
```

Comment:
Just a naming thing, but `mention_base_url` might be a bit more consistent with the other URLs defined here.